protocol/plasmid_test.xml

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <document>
-	<simulation name="biofilmCumul" outputfolder="../results" log="DEBUG"
-		comment="this is a file for testing purposes only." randomSeed="224323">
-		<timer stepSize="1.0" endOfSimulation="50.0" />
+	<simulation name="plasmid_test" outputfolder="../results" log="NORMAL"
+		comment="this is a file for testing purposes only." randomSeed="224323"
+		configuration="C:\Users\sanka\git\iDynoMiCS-2\config\alternative_colouring.cfg">
```

Comment:
Please also update this line, use relative paths to ensure the protocol works correctly on other systems.